src/components/Tabs/Tabs.jsx

```diff
@@ -1,28 +1,32 @@
-export const Tabs = ({ tabs, selectedTab, onTabSelected }) => (
-  <div data-cy="TabsComponent">
-    <div className="tabs is-boxed">
-      <ul>
-        {tabs.map(tab => (
-          <li
-            key={tab.id}
-            className={tab.id === selectedTab.id ? 'is-active' : null}
-            data-cy="Tab"
-          >
-            <a
-              href={`#${tab.id}`}
-              data-cy="TabLink"
-              onClick={() => onTabSelected(tab)}
+export const Tabs = ({ tabs, selectedTab, onTabSelected }) => {
+  const handleClick = tab => selectedTab.id !== tab.id && onTabSelected(tab);
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              key={tab.id}
+              className={tab.id === selectedTab.id ? 'is-active' : null}
```

Comment:
it's working, but better to use library classnames and insted of ? and : better to use operator &&